crates/bevy_app/src/plugin_group.rs

```diff
@@ -87,6 +96,7 @@ impl PluginGroupBuilder {
         self
     }
 
+    /// Disables a [`Plugin`]
```

Comment:
```suggestion
    /// Disables a [`Plugin`], preventing it from being add to the `App` with the rest of the [`PluginGroup`].
```

The precise meaning here is important, and can be a bit surprising.